admin-dev/themes/new-theme/js/pages/order/merge-shipment-manager.ts

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+import Router from '@js/components/router';
+import OrderViewPageMap from './OrderViewPageMap';
+
+export default class MergeShipmentManager {
+  private formRoute = 'admin_orders_shipment_get_merge_form';
+
+  private shipmentId: number|null = null;
+
+  private orderId: number|null = null;
+
+  private router = new Router();
+
+  constructor() {
+    this.initMergeShipmentEventHandler();
+  }
+
+  initMergeShipmentEventHandler(): void {
+    const mainDiv = document.querySelector(OrderViewPageMap.mainDiv);
+
+    if (!mainDiv) {
+      throw new Error('Main container not found');
```

Comment:
Can you please provide the details on the consequences of the missing container?

Something like "The feature to merge shipment cannot be initialized"